package.json

```diff
@@ -15,7 +15,8 @@
   ],
   "scripts": {
     "test": "jest --coverage --no-cache",
-    "build": "npm run bundle && npm run minify",
+    "build": "npm run build-hyperapp && npm run bundle && npm run minify",
```

Comment:
Take this back out when 1.0.0 is published and we don't need to build from GitHub.

Reply:
Done.